internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/model/repository/job/job_create.go

```diff
@@ -0,0 +1,33 @@
+package job
+
+import (
+	"context"
+
+	"github.com/keboola/go-utils/pkg/deepcopy"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop/op"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/model"
+)
+
+// Create a new stream Job.
+//
+// - If the Job already exists, the ResourceAlreadyExistsError is returned.
+func (r *Repository) Create(input *model.Job) *op.AtomicOp[model.Job] {
+	k := input.JobKey
+	var created model.Job
+	return op.Atomic(r.client, &created).
+		// Entity must not exist
+		Read(func(ctx context.Context) op.Op {
+			return r.MustNotExist(k)
+		}).
+		// Create
+		Write(func(ctx context.Context) op.Op {
+			// Create
+			created = deepcopy.Copy(*input).(model.Job)
+			return r.save(&created)
+		}).
+		// Update the input entity, it the operation is successful
```

Comment:
Looks like a typo? Should be `if` instead of `it`. I assume you copied this from elsewhere though...

Reply:
defdd2ca90ceeb940fad97242b874c87b9625a08